README.md

```diff
@@ -57,37 +57,43 @@ Everything required to connect and make API calls is located in a module respect
 You will have to provide a your own API Token from the [Authzed dashboard] in place of `t_your_token_here_1234567deadbeef` in the following example:
 
 ```js
-const authzed = require('@authzed/authzed-node/v0');
+import { v1 } from '@authzed/authzed-node';
```

Comment:
Can you add a link for the v1 api to the list under "Supported client API versions" above? The link can point to `https://docs.authzed.com/reference/api#authzedapiv1`

Reply:
Done